src/mlpack/methods/cf/cf_model.hpp

```diff
@@ -100,7 +100,7 @@ class CFWrapper : public CFWrapperBase
             const size_t numUsersForSimilarity,
             const size_t rank,
             const size_t maxIterations,
-            const size_t minResidue,
+            const double minResidue,
```

Comment:
if the value does not require high precision, I would prefer it to be float, but up to you

Reply:
In this case it actually should remain as `double` because some users may want a very small minimum residue.